Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -1044,13 +1045,23 @@ void MainWindow::NetPlayInit()
 {
   m_netplay_setup_dialog = new NetPlaySetupDialog(this);
   m_netplay_dialog = new NetPlayDialog;
+  m_netplay_dialog = new NetPlayDialog(this);
```

Comment:
This line shouldn't be here, it's a duplicate (which actually leaks a NetPlayDialog) and we construct without a parent intentionally.

Reply:
I'm not sure how that happened. I don't remember ever adding that.

Reply:
Probably a leftover from a rebase conflict.